pkg/app/api/grpcapi/web_api.go

```diff
@@ -653,8 +654,12 @@ func (a *WebAPI) GenerateApplicationSealedSecret(ctx context.Context, req *webse
 		return nil, status.Error(codes.FailedPrecondition, "The piped does not contain the encryption configuration")
 	}
 
-	var enc encrypter
+	data := req.Data
```

Comment:
should this `data` be used on [L678](https://github.com/pipe-cd/pipe/pull/1286/files#diff-de965e19dc71ecfcaec8cde1d2deebcc289b8b1e55db8ead99e2b5e4c03ebaf0R678) 👀

Reply:
Nice catch! Thank you.
Fixed!